src/python/testwatcher.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python2 -u
```

Comment:
Note here. The -u flag cannot be used on a shebang with env due to posix specification. It will either have to be kept in its original form, or be passed to the interpreter in some other way.